src/Tools/IdeCoreBenchmarks/NavigateToBenchmarks.cs

```diff
@@ -112,8 +113,72 @@ public void IterationCleanup()
             _workspace = null;
         }
 
+        // [Benchmark]
+        public async Task RunSerialIndexing()
+        {
+            Console.WriteLine("start profiling now");
+            // Thread.Sleep(10000);
```

Comment:
```suggestion
```

Reply:
so i find these tweaks helpful so that i can do things like run perfview sxs during certain parts of the run.  these tests really exist for me to drill down into issues, so i cater them to my needs :)